backend/src/main/java/pl/edu/agh/apdvbackend/services/GroupService.java

```diff
@@ -73,7 +73,8 @@ public Response<AboutGroupResponseBody> addGroup(
         return Response.withOkStatus(addNewGroup.execute(addGroupRequestBody));
     }
 
-    public Response<List<ShortUserResponseBody>> getNotMembersOfTheGroup(Long groupId) {
+    public Response<List<ShortUserResponseBody>> getNotMembersOfTheGroup(
```

Comment:
zmien nazwe na getNonMembersOfTheGroup

Reply:
ale no to takie czepialstwo

Reply:
Trzeba było nie mergować, to bym zmienił XD